tests/Omnipay/Adyen/Message/PurchaseRequestTest.php

```diff
@@ -94,6 +155,7 @@ public function testGetSetAllowedMethods()
 
     public function testGetEndpoint()
     {
-        $this->assertSame($this->request->getEndpoint(), 'https://test.adyen.com/hpp/pay.shtml');
+        $this->assertSame($this->request->getEndpoint(),
```

Comment:
@cduran433 please add a new line after the opening `(`, and put the closing `);` on a new line.